src/components/Checkbox/Checkbox.js

```diff
@@ -111,35 +114,42 @@ const CheckboxInput = styled('input')`
 
 const CheckboxLabel = styled('label')`
   ${labelBaseStyles}
-  ${labelCheckedStyles}
   ${labelDisabledStyles}
   ${labelInvalidStyles}
 `;
 
+const CheckboxWrapper = styled('div')`
+  ${checkboxWrapperBaseStyles};
+`;
+
 /**
  * Checkbox component for forms.
  */
-const Checkbox = ({ onToggle, children, id: customId, ...props }) => {
+const Checkbox = ({ onChange, children, id: customId, ...props }) => {
   const id = customId || uniqueId('checkbox_');
   return (
-    <Fragment>
-      <CheckboxInput id={id} onClick={onToggle} type="checkbox" {...props} />
+    <CheckboxWrapper>
+      <CheckboxInput id={id} onClick={onChange} type="checkbox" {...props} />
       <CheckboxLabel htmlFor={id} {...props}>
         {children}
       </CheckboxLabel>
-    </Fragment>
+    </CheckboxWrapper>
   );
 };
 
 Checkbox.propTypes = {
   /**
    * Controles/Toggles the checked state.
    */
-  onToggle: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
   /**
    * Value string for input.
    */
-  value: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.bool,
+    PropTypes.array
```

Comment:
When would this be an array?

Reply:
When using final form, there are two different behaviors:

- Do say what the value is, and final form uses booleans.
- Declare it, and final form adds the value to an array.

https://github.com/final-form/react-final-form#value-any

Reply:
I see. I'm a bit confused how this works exactly — would need to see this in an example.
I trust you've tested this? :)